test.sh

```diff
@@ -16,4 +16,9 @@
 
 # prow calls this file currently, but we can just use `make test` to test
 # the set of things we want.
-make test
+CHECK_DOCS_ONLY=$("test/check_docs_only.sh")
```

Comment:
How do you test this one since the PR https://github.com/kubernetes-sigs/kubebuilder/pull/2608, which has the script is not merged yet? Could you please add on the first comment the steps used to do the tests and its results?

Reply:
Also, how it can work and passing in the tests if we do not have the script yet?